### DISCUSSION TITLE
Sidebar "Recently Updated"  is NOT working in my Blog

### DISCUSSION THREAD

**Discussion by yesterz** [cotes2020/jekyll-theme-chirpy#1316] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

LIVE here -> https://github.com/yesterz/yesterz.github.io
It's not displaying as expected. 
I've tried to troubleshoot this issue, but so far, I haven't been able to resolve it.

### Operations you have already tried

Delete the file Gemfile.lock, then copy the .gitignore from [jekyll-theme-chirpy](https://github.com/cotes2020/jekyll-theme-chirpy) to your project and commit.

**Comment by cotes2020** [comment, marked as answer]
Follow the [tutorial](https://chirpy.cotes.page/posts/getting-started/#deployment), don't create `.github/workflows/jekyll.yml` yourself if you don't know Jekyll and GitHub Actions.

**Reply by yesterz** [reply]
**Thank you** for your prompt and effective help in solving the problem. **It works!**
`rm .github/workflows/jekyll.yml`
And mv from [cotes2020](https://github.com/cotes2020)/[chirpy-starter](https://github.com/cotes2020/chirpy-starter).github/workflows/pages-deploy.yml to its original place.
十分感谢！Orz.
